automatic/gimp/tools/chocolateyInstall.ps1

```diff
@@ -11,4 +13,11 @@ $packageArgs = @{
   validExitCodes = @(0)
 }
 
+Try {
+    Invoke-WebRequest -UseBasicParsing -Method HEAD -Uri $packageArgs.url | Out-Null
```

Comment:
Using `Invoke-WebRequest` can not be used.
Packages requires compatibility with PSv2, while `Invoke-WebRequest` was not added until PSv3.

Additionally, using this cmdlet would not behave correctly if the user is behind a proxy.

Possibly solutions would be to use [`Get-WebHeaders`](https://docs.chocolatey.org/en-us/create/functions/get-webheaders) or using [`Get-ChocolateyWebFile`](https://docs.chocolatey.org/en-us/create/functions/get-chocolateywebfile) (If it fails switch the URL and try again)

Reply:
Whoops, I forgot that it was a v3 addition. I've switched to using `Get-WebHeaders`.